configs/foreman/1.24.yaml

```diff
@@ -15,7 +15,7 @@
   - foreman-client-1.24-sles11
   - foreman-client-1.24-sles12
 :repos: []
-:gpg_key: 7B5B366A
+:gpg_key: 5AA9BCAD
```

Comment:
@zjhuntin this seems to be an oversight in the 1.24 config?

Reply:
It is, Something I needed to change, but forgot.